artemis-protocols/artemis-proton-plug/src/main/java/org/proton/plug/context/server/ProtonServerSessionContext.java

```diff
@@ -60,6 +61,9 @@ public int serverDelivery(Object message, Object consumer, int deliveryCount) th
 
    public void addTransactionHandler(Coordinator coordinator, Receiver receiver) {
       ProtonTransactionHandler transactionHandler = new ProtonTransactionHandler(sessionSPI);
+
+      coordinator.setCapabilities(Symbol.getSymbol("[\"amqp:local-transactions\",\"amqp:multi-txns-per-ssn\",\"amqp:multi-ssns-per-txn\"]"));
```

Comment:
I believe the Capabilities value should be an array of symbols containing the applicable individual capability values, rather than a CSV encoding of them.